kubeflow/trainer/rhai/instrumentation/traininghub_progression_test.py

```
@@ -0,0 +1,293 @@
+    algorithm = test_case.config["algorithm"]
+
+    algorithm_metadata = {
+        "name": algorithm,
+        "metrics_file_pattern": f"{algorithm}_metrics*.jsonl",
+        "metrics_file_rank0": f"{algorithm}_metrics_rank_0.jsonl",
+    }
+
+    # Verify the module creates the instrumentation successfully
+    apply_fn, handler_cls = create_traininghub_progression_instrumentation(
+        algorithm_metadata=algorithm_metadata,
+        ckpt_output_dir="/tmp/checkpoints",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/checkpoints"

(S108)